scripts/install.py

```diff
@@ -0,0 +1,3147 @@
+#!/usr/bin/env python3
+"""Cross-platform Spyglass installer.
```

Comment:
I count 50 individual functions, which is a lot of cognitive load to manage. Is there any way to group some subset into classes to silo responsibilities?